configs/katello/nightly.yaml

```diff
@@ -78,3 +99,47 @@
         - rhpkg-simple
         - rpm-build
         - shadow-utils
+  - name: katello-pulpcore-nightly-el8
+    based_off: null
+    arches:
+     - x86_64
+    build_target: katello-pulpcore-nightly-el8-build
+    external_repos:
+      - centos8-baseos
+      - centos8-appstream
+    build_groups:
```

Comment:
Do we need to specify this for the katello build tag above?

Reply:
I don't think so.  I followed the pattern with the prior katello el7 build tag, and it doesn't seem that it was there.

This could be due to an environment not entirely fleshed out as well.  I'll double check today.

Reply:
I think we never did that because it was already built out and for releases we often 'copied'. But for fresh tags, I find it worthwhile to specify the build tag components exactly as you've done here.

Reply:
So, after digging into it, and looking at katello-nightly-rhel7 on koji... it looks like the build_groups were set not by toolbelt, but manually somehow.  (it's not inheriting it's build groups)

Will add it to el8